src/main/java/com/epam/aidial/core/controller/DeploymentPostController.java

```diff
@@ -57,6 +57,7 @@ public class DeploymentPostController {
 
     private final Proxy proxy;
     private final ProxyContext context;
+    private boolean unregisterTrace;
```

Comment:
Please save trace id instead of flag and use it in unregister.

Reply:
fixed